corehq/apps/users/views/mobile/users.py

```diff
@@ -1497,18 +1506,41 @@ def user_id(self):
     @memoized
     def form(self):
         if self.request.method == 'POST':
-            return MobileWorkerAccountConfirmationForm(self.request.POST)
+            return MobileWorkerAccountConfirmationBySMSForm(self.request.POST)
         else:
-            if not self.is_invite_valid():
-                raise ValidationError("Invite has expired.")
-            return MobileWorkerAccountConfirmationForm(initial={
+            return MobileWorkerAccountConfirmationBySMSForm(initial={
                 'username': self.user.raw_username,
                 'full_name': self.user.full_name,
-                'email': self.user.email,
+                'email': f"{self.user.raw_username}@{self.domain}.{settings.HQ_ACCOUNT_ROOT}",
```

Comment:
Why store the HQ username as the email address instead of just leaving it blank?

Reply:
That should also be fine. Before removing the email field I was thinking to populate email input with this value on form load but kept the same variable.